etna/transforms/timestamp/holiday.py

```diff
@@ -52,6 +85,7 @@ def __init__(self, iso_code: str = "RUS", mode: str = "binary", out_column: Opti
         self._mode = HolidayTransformMode(mode)
         self.holidays = holidays.country_holidays(iso_code)
         self.out_column = out_column
+        self.freq = ""
```

Comment:
Let's set `self.freq = Optional[str] = None` here and in `_transform` let's check that the transform is fitted by looking at `self.freq` like it is done in many other transforms.